autocert/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: autocert
-version: 1.19.3
-appVersion: 0.19.3
+version: 0.19.7
```

Comment:
Is it OK to "downgrade" this to the `0.` release? Will it be seen as an update correctly in that case?

I've seen the mismatch between the chart and the app version before, and thought it was intentional.

Reply:
Fixed with 51b5de7